### ISSUE TITLE
Idea: Extend display of search results

### ISSUE THREAD

**Issue by stefandesu** [gbv/cocoda#697]
Currently, when searching for a concept inside a vocabulary, only the preferred label is shown in the result. Depending on how someone is using Cocoda, it might be helpful to show more detailed search results that include things like hierarchy. I could imagine to show those results in place of the ConceptDetails component or even extend it over the whole vertical space so that enough information can be shown.

This idea came from @DavidBRohrer telling me that he's still using a vocabulary's respective online search instead of Cocoda's search because it makes it easier to grasp the search results. What do you suggest should be shown in the search results? Should it maybe be configurable? Should there be a toggle to always use these "rich" search results?

**Comment by nichtich** [comment]
Scope notes and alternative labels are worth to include but inclusion of hierarchy likely depends on the vocabulary. Maybe vocabulary type can be used as hint? In any case we need to try out.

**Comment by guitarster** [comment]
BK: scope notes should be shown. Hierarchy is also important.
DDC: same as with BK. Additionally, Mappings to GND and "Registereinträge" are important.

**Comment by stefandesu** [comment]
Some suggestions and considerations (I'd appreciate feedback on these points):

- Space is a big issue here. I'd suggest to make the results wider (roughly half the screen width) and taller (maybe even almost to the bottom).
- If we have a deeply nested concept, showing all ancestors including their labels would take a LOT of space. Would it make sense to show only the top concept with label and the rest with notation only?
- Same with scope notes and Registereinträge, although here I would suggest showing one line and cutting it off at the end if necessary.
- What about alternative labels? That would be an additional line.
- There was the idea of keeping search as it as and only adding mouseovers with the detailed information, but then you'd still not get a good overview over the results.

**Comment by guitarster** [comment]
I would prefer mouseover; so it would be no problem to make the results as wide and tall as you mentioned. 

Ancestors: what about showing only the labels in one line (and not top to bottom)? Notations are not so important but the labels, so you can understand the context of the concept.

Yes, one line to show scope notes and Registereinträge seems to be the best solution. 

Alternative labels: do you mean synonymes "Syn.: Blut und blutbildende Organe" at BK 44.86 Hämatologie? This information is important to me.

**Comment by stefandesu** [comment]
> I would prefer mouseover; so it would be no problem to make the results as wide and tall as you mentioned.

Mouseover would indeed make it easier as we'd only need to show the details for a single result instead of all results. I thought you'd prefer to see as much information as possible on one screen, but I guess this is more about being able to (quickly) differentiate between certain results if it's not clear what is meant by only notation/label.

> Ancestors: what about showing only the labels in one line (and not top to bottom)? Notations are not so important but the labels, so you can understand the context of the concept.

I will experiment. However, I do feel like the notations should be shown as well.

> Yes, one line to show scope notes and Registereinträge seems to be the best solution.

👍

> Alternative labels: do you mean synonymes "Syn.: Blut und blutbildende Organe" at BK 44.86 Hämatologie? This information is important to me.

Exactly. It is listed under Labels -> Alternative Labels in Cocoda.

**Comment by guitarster** [comment]
Yes, in general I think you are right that a result list with detailed Information to every result would be better than mouseover to one result. I thought mouseover would be a good compromise.
